src/services/housing.js

```diff
@@ -55,12 +58,11 @@ import './user';
  * @return {Promise.<Boolean>} True if the user is authorized to view the housing application staff page
  */
 const checkHousingStaff = async () => {
-  return false; //! DEBUG
-  // try {
-  //   return await http.get(`housing/staff`);
-  // } catch {
-  //   return false;
-  // }
+  try {
+    return await http.get(`housing/staff`);
+  } catch {
+    return false;
+  }
 };
 
 /**
```

Comment:
Also, the comment on this service could use some clarification that it is getting the ID of an application _from this session_.

Reply:
Maybe "semester", since "session" doesn't seem to have much relevance on the frontend, as far as I've seen so far.

Reply:
Okay, sure :)